src/event-stream/event-server.ts

```diff
@@ -728,105 +729,104 @@ export async function startEventServer(opts: {
       .json({ status: 'ready', msg: 'API event server listening for core-node POST messages' });
   });
 
-  app.post(
-    '*',
-    asyncHandler(async (req, res, next) => {
-      const eventPath = req.path;
-      let payload = JSON.stringify(req.body);
-      await messageHandler.handleRawEventRequest(eventPath, req.body, db);
-      if (logger.isDebugEnabled()) {
-        // Skip logging massive event payloads, this _should_ only exclude the genesis block payload which is ~80 MB.
-        if (payload.length > 10_000_000) {
-          payload = 'payload body too large for logging';
-        }
-        logger.debug(`[stacks-node event] ${eventPath} ${payload}`);
```

Comment:
Any reason for removing this log?

Reply:
@rafaelcr I took your comment as you wanted this removed. Did I read your comment correctly?

Reply:
@BLuEScioN I think you'll just need to move it to the `handleRawEventRequest` function you created below